s3o_import.py

```diff
@@ -266,13 +266,21 @@ def load(self, fhandle, offset, material):
             except AttributeError:
                 # Blender < 2.80
                 bpy.context.scene.objects.link(self.ob)
-            bpy.context.scene.update()
+            try:
+                bpy.context.scene.update()
+            except AttributeError:
+                # Blender > 2.80
+                # TODO: don't know how to update the scene!

```

Comment:
I've included this here as the attribute error is thrown in 2.80, but it loads and shows fine, so I'm... eh, maybe 90% sure this isn't actually needed in 2.80 anymore?  If anyone knows, please let me know for certain.

Reply:
Prolly!